EdvClient.js

```diff
@@ -546,8 +549,13 @@ export class EdvClient {
     });
     // send request
     const {httpsAgent} = this;
+
     const response = await axios.post(url, query, {headers, httpsAgent});
+    if(count) {
```

Comment:
it might be better (as count should be a boolean in all cases) to do
```
if(count === true) {
  return response.data;
}
```

and thank you for doing this.